master/internal/db/postgres_rbac.go

```diff
@@ -0,0 +1,10 @@
+package db
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// ErrNotEnoughPermissions is returned when a user
+// does not have permissions required.
+var ErrNotEnoughPermissions = status.Error(codes.PermissionDenied, "access denied")
```

Comment:
Some thoughts:
- This doesn't seem like a DB concern; it seems like an API concern. So IMO it should go in an API-related package rather than a DB one.
- This isn't RBAC-specific (as evidenced by it being called by the basic authz implementation), so I recommend a name like `ErrAccessDenied` to avoid talking about the RBAC-specific implementation detail of permissions.
- We should avoid creating files in OSS that might need to be created in EE to minimize rebasing headaches.